cratedb_toolkit/io/processor/kinesis_lambda.py

```diff
@@ -22,25 +34,70 @@
 import logging
 import os
 import sys
-import typing as t
 
 import sqlalchemy as sa
+from commons_codec.exception import UnknownOperationError
+from commons_codec.model import ColumnTypeMapStore
+from commons_codec.transform.aws_dms import DMSTranslatorCrateDB
 from commons_codec.transform.dynamodb import DynamoCDCTranslatorCrateDB
+from sqlalchemy.util import asbool
 
-logger = logging.getLogger(__name__)
+LOG_LEVEL: str = os.environ.get("LOG_LEVEL", "INFO")
+USE_BATCH_PROCESSING: bool = asbool(os.environ.get("USE_BATCH_PROCESSING", "false"))
+ON_ERROR: str = os.environ.get("ON_ERROR", "exit")
+SQL_ECHO: bool = asbool(os.environ.get("SQL_ECHO", "false"))
 
-# TODO: Control using environment variable.
-logger.setLevel("INFO")
+MESSAGE_FORMAT: str = os.environ.get("MESSAGE_FORMAT", "unknown")
+COLUMN_TYPES: str = os.environ.get("COLUMN_TYPES", "")
+CRATEDB_SQLALCHEMY_URL: str = os.environ.get("CRATEDB_SQLALCHEMY_URL", "crate://")
+CRATEDB_TABLE: str = os.environ.get("CRATEDB_TABLE", "default")
```

Comment:
That's the only deviation compared to the upstream variant: There, those variables are called `SINK_SQLALCHEMY_URL` and `SINK_TABLE`.